### DISCUSSION TITLE
NetBox no longer working after update from 3.4.10 to 3.5.0

### DISCUSSION THREAD

**Discussion by victoriaronja** [netbox-community/netbox#13192] [Help Wanted!] [answered]
Hello everyone,

I have tried updating my testing environment from 3.4.10 to 3.5.0 but when trying to access NetBox after running the upgrade script, I am met with the following error message:

```
<class 'ConnectionResetError'>

[Errno 104] Connection reset by peer

Python version: 3.8.16
NetBox version: 3.5.0
```

Could anyone help me out here? Thanks in advance.

**Comment by candlerb** [comment, marked as answer]
You should go straight for 3.5.6 - that rolls up a bunch of bug fixes.

My *guess* is that you are being caught out by the new dashboard and the widget which makes outbound connections to download a list of Netbox news items. When introduced, this broke people in environments where the Netbox server is not allowed to make outbound connections, but fixes were made in later versions of v3.5.x so that this wouldn't stop Netbox from running.

If that's not it, then get a full backtrace (either set `ADMINS` and `EMAIL` in configuration.py, or set `DEBUG=True` to get it via the web interface)
